bibliopixel/main/demo.py

```diff
@@ -0,0 +1,60 @@
+import webbrowser
+
+from bibliopixel.drivers.SimPixel import DriverSimPixel
+from bibliopixel import LEDCircle, LEDMatrix, layout
+
+SIMPIXEL_URL = 'http://beta.simpixel.io'
+IMPORT_ERROR_TEXT = """
+Please install the BiblioPixelAnimations library from here:
+
+https://github.com/ManiacalLabs/BiblioPixelAnimations
+
+"""
+
+
+def bloom(args):
+    driver = DriverSimPixel(args.width * args.height)
+    led = LEDMatrix(driver, width=args.width, height=args.height)
+
+    from BiblioPixelAnimations.matrix.bloom import Bloom
+    return Bloom(led)
+
```

Comment:
Nothing.  :-D  It seems to run, but nothing comes up on the simpixel page!

Don't approve this, we can't check it in until it runs.  :-P

I don't remember where I got this.  I have yet to actually run a circle animation - suspect it's my fault but...